src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -31,39 +57,43 @@ public String todayDate(DateTimePart datePart) {
      *                   - 3-й элемент массива - день (число);
      */
     public Optional<LocalDate> getDate(Integer[] dateParams) {
-        return Optional.empty();
+        try {
+            return Optional.of(LocalDate.of(dateParams[0], dateParams[1], dateParams[2]));
+        } catch (RuntimeException e) {
+            return Optional.empty();
```

Comment:
let's move it out of catch block, we should not use try catch to build our logic